themes/bootstrap3/templates/ajax/status-full.phtml

```diff
@@ -43,6 +51,16 @@
           <span class="<?=$this->escapeHtmlAttr($statusClass)?>"><?=$this->transEsc($statusDescription, $statusDescriptionTokens)?></span>
       </td>
     </tr>
+
+    <?php foreach ($this->holdingsTextFieldNames as $holdingsTextField): ?>
+      <?php if ($item[$holdingsTextField]): ?>
+        <tr>
+          <td colspan="3" class="statusItemsHoldingsTextFields">
```

Comment:
Did you consider a `<th scope="row">...</th><td colspan="2">...</td>` pattern here?

I also notice that the existing `<th>` tags at the top lack the `scope="col"` attribute, which might improve accessibility... but mixing scopes in a single table is probably a bad thing, so maybe we need to rethink the whole approach.

In any case, just brainstorming here; as discussed earlier, we don't need to change the markup to get this PR merged, but it's something we should consider improving in a follow-up.

Reply:
No, I hadn't thought about this.  I wonder if this would roll into the issue @maccabeelevine created:

https://openlibraryfoundation.atlassian.net/browse/VUFIND-1707.

Reply:
Yes, happy to defer the cleanup of this to the resolution of that ticket. I'll make note of this conversation there.